go/simple/v1/example/ampli-example.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"ampli-example/ampli"
+	"github.com/amplitude/analytics-go/amplitude"
+	"github.com/joho/godotenv"
+	"log"
+	"os"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatalln("Error loading .env file")
+	}
+	apiKey := os.Getenv("AMPLITUDE_API_KEY")
+
+	// Initialize the Ampli instance with LoadOptions and LoadClientOptions
+	ampli.Client.Load(ampli.LoadOptions{
```

Comment:
I think, it looks simpler and better to use `ampli` insteaf of `ampli.Client`:
```
ampli.Load(ampli.LoadOptions{
	Client: ampli.LoadClientOptions{
		Configuration: amplitude.NewConfig(apiKey),
	},
})
defer ampli.Shutdown()

ampli.EventNoProperties("user_id", amplitude.EventOptions{})
```
If we need a new instance:
```
ampli2 := ampli.New()
ampli2.Load(ampli.LoadOptions{
	Client: ampli.LoadClientOptions{
		Configuration: amplitude.NewConfig(apiKey2),
	},
})
defer ampli2.Shutdown()

ampli2.EventNoProperties("user_id", amplitude.EventOptions{})
```

If we still want to use the singleton explicitely `ampli.Instance` looks better, I think. For example, `ampli.Instance.Client` and `ampli.Client.Client`

Reply:
+1

Reply:
To call `ampli.Load()` directly, I can write a wrapper like this:
```
var Client = Ampli{}

type Ampli struct {
}

func (a *Ampli) Load(options LoadOptions) {
}

func Load(options LoadOptions) {
	Client.Load(options)
}
```
1. Do you think the above implementation would be a good way?
2. I should add and expose a `New()` func for initialization. Will update it

Reply:
I was wrong. The wrapper doesn't work when it comes to `Identify()` func. We cannot declare `Identify()` func and `Identify` within the same package.

Reply:
Renamed default Ampli to `Instance`